JASP-Engine/JASP/R/regressionlogistic.R

```diff
@@ -195,28 +184,32 @@ RegressionLogistic <- function(jaspResults, dataset = NULL, options, ...) {
   estimatesTableBootstrap <- createJaspTable(gettext("Bootstrap Coefficients"))
   estimatesTableBootstrap$dependOn(optionsFromObject   = jaspResults[["modelSummary"]],
                                    options             = c("coeffEstimatesBootstrapping",
-                                                           "coeffEstimatesBootstrappingReplicates"))
+                                                           "coeffEstimatesBootstrappingReplicates",
+                                                           "coeffCI", "coeffCIOR", "stdCoeff", "oddsRatios", "coeffCIInterval",
+                                                           "robustSEOpt"))
   estimatesTableBootstrap$position <- 3
   estimatesTableBootstrap$showSpecifiedColumnsOnly <- TRUE
   
-  if (options$method == "enter") {
-    multimod   <- FALSE
-    paramtitle <- ""
-  } else {
-    multimod   <- TRUE
-    paramtitle <- gettext("Parameter")
-  }
-  
-  ciTitle <- gettextf("95%% bca%s Confidence interval", "\u002A")
-  
+  tmp <- .reglogisticEstimatesInfo(options, addBCA = TRUE)
+  ciTitle    <- tmp[["ciTitle"]]
+  seTitle    <- tmp[["seTitle"]]
+  multimod   <- tmp[["multimod"]]
+  paramtitle <- tmp[["paramtitle"]]
+
   if(options$method != "enter")
     estimatesTableBootstrap$addColumnInfo(name = "model", title = gettext("Model"),          type = "string", combine = TRUE)
   estimatesTableBootstrap$addColumnInfo(name = "param",   title = paramtitle,                type = "string")
   estimatesTableBootstrap$addColumnInfo(name = "est",     title = gettext("Estimate"),       type = "number", format="dp:3")
   estimatesTableBootstrap$addColumnInfo(name = "bias",    title = gettext("Bias"),           type = "number", format="dp:3")
-  estimatesTableBootstrap$addColumnInfo(name = "se",      title = gettext("Standard Error"), type = "number", format="dp:3")
-  estimatesTableBootstrap$addColumnInfo(name = "cilo",    title = gettext("Lower bound"),    type = "number", format="dp:3", overtitle = ciTitle)
-  estimatesTableBootstrap$addColumnInfo(name = "ciup",    title = gettext("Upper bound"),    type = "number", format="dp:3", overtitle = ciTitle)
+  estimatesTableBootstrap$addColumnInfo(name = "se",      title = seTitle,                   type = "number", format="dp:3")
+  if(options$stdCoeff)
+    estimatesTableBootstrap$addColumnInfo(name = "std",   title = gettextf("Standardized%s", "\u207A"), type = "number", format="dp:3")
```

Comment:
Good to add the footnote too, if you add the symbol for the footnote (not sure if people would look at different tables for the symbol)